docs/source/clients.rst

```diff
@@ -0,0 +1,104 @@
+============
+Clients
+============
+
+There are two client executables that can be used with hipBLAS. They are,
+
+1. hipblas-bench
+
+2. hipblas-test
+
+These two clients can be built by following the instructions at `Building and Installing hipBLAS github page <https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/docs/source/install.rst>`_. After building the hipBLAS clients, they can be found in the directory ``hipBLAS/build/release/clients/staging``.
+
```

Comment:
The link does not work. replace with 

These two clients can be built by following the instructions in the "Building and Installing" section.

Reply:
Looks like this link is still here and still doesn't link correctly.

Reply:
The link will start working once this PR is merged to develop branch.